terraform/modules/tf_stream_alert_cloudtrail/main.tf

```diff
@@ -1,8 +1,10 @@
-// StreamAlert CloudTrail
+// StreamAlert CloudTrail, also sending to CloudWatch Logs group
 resource "aws_cloudtrail" "streamalert" {
-  count                         = "${var.existing_trail ? 0 : 1}"
+  count                         = "${!var.existing_trail && var.send_to_cloudwatch ? 1 : 0}"
```

Comment:
Silly thing, but can we switch this order? It makes more sense in English to say `send_to_cloudwatch and not existing_trail` (also below)

Reply:
sure thing!